src/main/java/mate/academy/springboot/datajpa/controller/CategoryController.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.springboot.datajpa.controller;
+
+import mate.academy.springboot.datajpa.dto.request.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.response.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import mate.academy.springboot.datajpa.service.mapper.CategoryMapper;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController()
+@RequestMapping("/categories")
+public class CategoryController {
+    private final CategoryService categoryService;
+    private final CategoryMapper categoryMapper;
+
+    public CategoryController(CategoryService categoryService, CategoryMapper categoryMapper) {
+        this.categoryService = categoryService;
+        this.categoryMapper = categoryMapper;
+    }
+
+    @PostMapping
+    public CategoryResponseDto create(@RequestBody CategoryRequestDto requestDto) {
+        Category category = categoryMapper.toModel(requestDto);
+        return categoryMapper.toDto(categoryService.save(category));
+    }
+
+    @GetMapping("/{id}")
+    public CategoryResponseDto getById(@PathVariable Long categoryId) {
+        Category category = categoryService.getById(categoryId);
+        return categoryMapper.toDto(category);
+    }
+
+    @DeleteMapping("/{categoryId}")
```

Comment:
```suggestion
    @DeleteMapping("/{id}")
```
the same for other methods